Cubical/Categories/LocallySmall/Presheaf/Base.agda

```diff
@@ -0,0 +1,216 @@
+    SmallPresheaf→Presheaf P .F-id = P .SFunctor.F-id
+    SmallPresheaf→Presheaf P .F-seq {x = liftω x} {z = liftω z} f g =
+      P .SFunctor.F-seq f g
+      -- I'd like to use reind-filler reasoning principles
+      -- but SET.reind-filler requires many implicits to be filled in
+      -- ∙ (SET.≡out
+      --      {xᴰ = liftω (P .SFunctor.F-ob x)}
+      --      {yᴰ = liftω (P .SFunctor.F-ob z)} $
+      --      SET.reind-filler
+      --        {xᴰ = liftω (P .SFunctor.F-ob x)}
+      --        {yᴰ = liftω (P .SFunctor.F-ob z)}
+      --         refl _)
+      ∙ (sym $ transportRefl _)
+
+    Presheaf→SmallPresheaf : Presheaf (mkSmallCategory C) ℓP → SmallPresheaf ℓP
+    Presheaf→SmallPresheaf P .SFunctor.F-ob = λ z → F-ob P (liftω z) .lowerω
+    Presheaf→SmallPresheaf P .SFunctor.F-hom = P .F-hom
+    Presheaf→SmallPresheaf P .SFunctor.F-id = P .F-id
+    Presheaf→SmallPresheaf P .SFunctor.F-seq {x = x} {z = z}
+       f g =
+      P .F-seq f g
+      -- ∙ (SET.≡out
+      --      {xᴰ = P .F-ob (liftω x)}
+      --      {yᴰ = P .F-ob (liftω z)} $
+      --      sym $ SET.reind-filler
+      --              {xᴰ = P .F-ob (liftω x)}
+      --              {yᴰ = P .F-ob (liftω z)}
+      --              refl _)
+      ∙ (transportRefl _)
+
+    SmallPresheaf→Presheaf→SmallPresheaf : ∀ (P : SmallPresheaf ℓP) →
+      Presheaf→SmallPresheaf (SmallPresheaf→Presheaf P) ≡ P
+    SmallPresheaf→Presheaf→SmallPresheaf P =
+      SmallFunctor.Functor≡ (λ _ → refl) λ _ → refl
+
+    Presheaf→SmallPresheaf→Presheaf-F-ob :
+      ∀ (P : Presheaf (mkSmallCategory C) ℓP) →
+      (c : mkSmallCategory C .SmallCategory.small-ob) →
+      P .F-ob (liftω c) .lowerω ≡ Presheaf→SmallPresheaf P .SFunctor.F-ob c
+    Presheaf→SmallPresheaf→Presheaf-F-ob P c = refl
+
+    Presheaf→SmallPresheaf→Presheaf-F-hom :
+      ∀ (P : Presheaf (mkSmallCategory C) ℓP) →
+         {c c' : mkSmallCategory C .SmallCategory.small-ob} →
+      P .F-hom {x = liftω c} {y = liftω c'} ≡ Presheaf→SmallPresheaf P .SFunctor.F-hom
+    Presheaf→SmallPresheaf→Presheaf-F-hom P = refl
+
+open Functor
+module _
+  (C : SmallCategory ℓC ℓC')
+  (c : C .SmallCategory.small-ob)
+  where
+  private
+    module C = SmallCategory C
+
+  _[-,_] : Presheaf C ℓC'
+  _[-,_] .F-ob c' = liftω (C.Hom[ c' , liftω c ] , C.isSetHom)
+  _[-,_] .F-hom f g = f C.⋆ g
+  _[-,_] .F-id = funExt λ g → C.⋆IdL _
+  _[-,_] .F-seq {x = x} {y} {z} f g =
+    (funExt λ h → C.⋆Assoc g f h)
+    ∙ (SET.≡out
+         {xᴰ = liftω (C.Hom[ x , liftω c ] , C.isSetHom)}
+         {yᴰ = liftω (C.Hom[ z , liftω c ] , C.isSetHom)}
+         $
+         SET.reind-filler
+           {xᴰ = liftω (C.Hom[ x , liftω c ] , C.isSetHom)}
+           {yᴰ = liftω (C.Hom[ z , liftω c ] , C.isSetHom)}
+           refl λ h → g C.⋆ (f C.⋆ h))
+
+open SmallFibNatTrans
+module _
+  {C : SmallCategory ℓC ℓC'}
+  where
+  private
+    module C = SmallCategory C
+  open Categoryᴰ
+
+  よ : Functor  C.cat (∫C (PRESHEAF C))
```

Comment:
This terminology is inconsistent. In the existing library we call this YO in `Categories.Yoneda` or `YONEDA` in `Categories.Yoneda.More`